launch_yaml/launch_yaml/parser.py

```diff
@@ -57,4 +57,4 @@ def load(
     @classmethod
     def get_file_extensions(cls) -> Set[Text]:
         """Return the set of file extensions known to this parser."""
-        return {'launch.yaml', 'launch.yml', 'yaml', 'yml'}
+        return {'launch.yaml', 'launch.yml', 'yaml', 'yml', 'launch'}
```

Comment:
@christophebedard meta: one that we probably missed in #516 is that e.g. `yaml` already includes `launch.yaml` making the latter redundant.

Reply:
yeah I noticed the redundancy too :laughing:

I simply thought it didn't really change anything in practice and it acts as an explicit list of supported file extensions even if it's redundant, but I can remove the redundant ones if you want. Want me to remove them?

Reply:
No biggie. We can remove them in a follow-up PR.